post_office/tests/test_backends.py

```diff
@@ -94,7 +94,7 @@ def test_backend_attachments(self):
         message = EmailMessage('subject', 'body', 'from@example.com',
                                ['recipient@example.com'])
 
-        message.attach('attachment.txt', 'attachment content')
+        message.attach('attachment.txt', b'attachment content')
```

Comment:
@selwin Please check here carefully

Reply:
All is ok
https://github.com/django/django/blob/e3c2fae4cd549a6aeefcf7b217ba4b173489e18c/tests/mail/tests.py#L491